meteor/client/ui/ClockView/Timediff.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import ClassNames from 'classnames'
+import { RundownUtils } from '../../lib/rundown'
+
+export const Timediff = class Timediff extends React.Component<{ time: number }> {
+	render() {
+		const time = -this.props.time
+		const isNegative = Math.floor(time / 1000) > 0
+		const timeString = RundownUtils.formatDiffToTimecode(time, true, false, true, false, true, '', false, true) // @todo: something happened here with negative time
+		// RundownUtils.formatDiffToTimecode(this.props.displayTimecode || 0, true, false, true, false, true, '', false, true)
```

Comment:
What about this TODO then? Is it something that should be done, or should the commented out section be deleted?

Reply:
Unfortunately, I don't know what that relates to, since it's a leftover from the code before the refactoring. As far as I know this view works the way it should, so I'm just going to remove this commented out block.